### DISCUSSION TITLE
有道词典笔第三代（YDP031）成功刷入词典笔os3.4.3

### DISCUSSION THREAD

**Discussion by youoshin0614** [PenUniverse/PenMods#239] [奇技淫巧]
![IMG_1239](https://github.com/PenUniverse/PenMods-release/assets/68409800/50057a06-f5d7-4168-8d10-6bb07c56f93f)
![IMG_1238](https://github.com/PenUniverse/PenMods-release/assets/68409800/9d20c8f4-561c-43be-9ae3-c35a325e52c5)
朋友的返厂维修后固件更新成词典笔os3.4.3
提取system_a后发现可以直接刷进最新版本的ydp031中
（其他分区应该没变，可以正常使用）
估计是工厂还在做适配，里面还有开发者模式

**Comment by chw2333** [comment]
求镜像

**Reply by youoshin0614** [reply]
[https://www.123pan.com/s/8KCzVv-AqdVd.html提取码:W2Md](https://www.123pan.com/s/8KCzVv-AqdVd.html%E6%8F%90%E5%8F%96%E7%A0%81:W2Md)
升级到最新系统后刷入system_A和system_B即可
（由于不是常规版本，没法用抓包，所以直接刷进去就行）

**Reply by wlw333554** [reply]
怎么刷入啊？

**Comment by SkySight-666** [comment]
大佬求固件

**Reply by youoshin0614** [reply]
https://www.123pan.com/s/8KCzVv-AqdVd.html提取码:W2Md
升级到最新系统后刷入system_A和system_B即可
（由于不是常规版本，没法用抓包，所以直接刷进去就行）

**Reply by SkySight-666** [reply]
谢谢佬

**Comment by AronnaxM** [comment]
不知道此版本是否支持YDP038 CHN PRO？

**Comment by YumeSemi** [comment]
我也是ydp031，但是刷进去卡开机，估计其他分区也有不同

**Comment by bpyqd** [comment]
YDP031成功刷入，但是应用商店用不了，显示更新列表为空

**Reply by youoshin0614** [reply]
可能是网易那边关掉了（？，我之前用着没问题

**Reply by SkySight-666** [reply]
> YDP031成功刷入，但是应用商店用不了，显示更新列表为空

检查是否使用了测试地址

**Reply by bpyqd** [reply]
> > YDP031成功刷入，但是应用商店用不了，显示更新列表为空
> 
> 检查是否使用了测试地址

我确定并没有打开测试地址，经测试我不管是打开还是关闭测试地址都是无法访问的

**Reply by bpyqd** [reply]
> 可能是网易那边关掉了（？，我之前用着没问题

有可能，我看别人的p3刷了词典笔os后也不能访问

**Reply by SkySight-666** [reply]
> > > YDP031成功刷入，但是应用商店用不了，显示更新列表为空
> > 
> > 
> > 检查是否使用了测试地址
> 
> 我确定并没有打开测试地址，经测试我不管是打开还是关闭测试地址都是无法访问的

抓个包看看？

**Reply by bpyqd** [reply]
> > > > YDP031成功刷入，但是应用商店用不了，显示更新列表为空
> > > 
> > > 
> > > 检查是否使用了测试地址
> > 
> > 
> > 我确定并没有打开测试地址，经测试我不管是打开还是关闭测试地址都是无法访问的
> 
> 抓个包看看？

已经卖了😭

**Comment by onear233** [comment]
刷完卡开机咋办

**Comment by Zecci9** [comment]
YDP031 std怎么刷回以前的版本啊

**Reply by bpyqd** [reply]
升级了这个词典笔os想回到原来的系统？
discussion里有备份的固件，刷入即可

**Comment by wlw333554** [comment]
我不到啊

________________________________
发件人: Zecc_FY ***@***.***>
发送时间: 2025年5月3日 13:02
收件人: PenUniverse/PenMods-release ***@***.***>
抄送: wlw333554 ***@***.***>; Comment ***@***.***>
主题: Re: [PenUniverse/PenMods-release] 有道词典笔第三代（YDP031）成功刷入词典笔os3.4.3 (Discussion #239)


YDP031 std怎么刷回以前的版本啊

―

**Comment by wlw333554** [comment]
如何刷我都不知道

________________________________
发件人: Wu Liwen ***@***.***>
发送时间: 2025年6月27日 11:02
收件人: PenUniverse/PenMods-release ***@***.***>; PenUniverse/PenMods-release ***@***.***>
抄送: Comment ***@***.***>
主题: 回复: [PenUniverse/PenMods-release] 有道词典笔第三代（YDP031）成功刷入词典笔os3.4.3 (Discussion #239)

我不到啊

________________________________
发件人: Zecc_FY ***@***.***>
发送时间: 2025年5月3日 13:02
收件人: PenUniverse/PenMods-release ***@***.***>
抄送: wlw333554 ***@***.***>; Comment ***@***.***>
主题: Re: [PenUniverse/PenMods-release] 有道词典笔第三代（YDP031）成功刷入词典笔os3.4.3 (Discussion #239)


YDP031 std怎么刷回以前的版本啊

―

**Comment by wlw333554** [comment]
请你教教我刷到os3.4.3
________________________________
发件人: Zecc_FY ***@***.***>
发送时间: 2025年5月3日 13:02
收件人: PenUniverse/PenMods-release ***@***.***>
抄送: wlw333554 ***@***.***>; Comment ***@***.***>
主题: Re: [PenUniverse/PenMods-release] 有道词典笔第三代（YDP031）成功刷入词典笔os3.4.3 (Discussion #239)


YDP031 std怎么刷回以前的版本啊

―

**Reply by bpyqd** [reply]
可以参考这个discussion里的b站专栏，里面有一个二代加强版升级满分版的教程，对照教程刷入即可

**Comment by wenmang123** [comment]
我该怎么刷入呢

**Comment by wenmang123** [comment]
请你教教我可以吗
